render/wrappedtext.go

```diff
@@ -36,14 +36,23 @@ type WrappedText struct {
 	Width       int
 	LineSpacing int
 	Color       color.Color
+	Alignment   string
 }
 
 func (tw WrappedText) Paint(bounds image.Rectangle, frameIdx int) image.Image {
 	face := Font[DefaultFontFace]
 	if tw.Font != "" {
 		face = Font[tw.Font]
 	}
-
+	// Text alignment
+	alignment := gg.AlignLeft
+	if tw.Alignment == "center" {
+		alignment = gg.AlignCenter
+	} else if tw.Alignment == "right" {
+		alignment = gg.AlignRight
+	} else {
```

Comment:
This branch is redundant since you set the value to AlignLeft before the two if statements.

Reply:
I assume you mean the `else` statement. you need the first two conditions. I mostly did it for completion but you have to initialize the variable some how so I left it as is for completion.

Reply:
I went ahead though and removed it. I probably never changed it from my initial submission